lib/scaffold.dart

```diff
@@ -633,7 +639,8 @@ class BackdropScaffoldState extends State<BackdropScaffold>
 
   Widget _buildBody(BuildContext context) {
     return WillPopScope(
-      onWillPop: () => _willPopCallback(context),
+      onWillPop: (() => _willPopCallback(context) as Future<bool>)
+          as Future<bool> Function()?,
```

Comment:
Can be changed if we are able to change return type of `_willPopCallback` to `Future<bool>`.

Reply:
Resolved in https://github.com/fluttercommunity/backdrop/commit/2f4010e5619ad2089290835c3e40b7048594943f.

Reply:
Yeah, not returning "null" in `_willPopCallback`, makes sense.

I have pushed the changes in https://github.com/fluttercommunity/backdrop/pull/86/commits/c77c561f2f754576242980531f8e08710632e2a0.

Resolve this, if you think they are ok?